osi_route.proto

```diff
@@ -0,0 +1,109 @@
+syntax = "proto3";
+
+option optimize_for = SPEED;
+
+import "osi_common.proto";
+
+package osi3;
+
+//
+// \brief A route in the road network
+//
+// A route is a planned or suggested path for an agent to travel from one location
+// to another within the road network. It is composed of a list of route segments,
+// which form a continuous path through the road network and should be traversed
+// in the order they are listed.
+//
+// ## Example
+//
+// The example below shows the \link Route route\endlink of a vehicle.
+//
+// \image html OSI_Planned_Route.png "Route" width=850px
+//
+// The route is composed of three \link RouteSegment route segments\endlink RS1-3,
+// each indicated by a yellow outline. Two of the route segments
+// (RS2 and RS3) only contain a single \link LogicalLaneSegment logical lane segment\endlink
+// (highlighted in blue), while RS1 is composed of three
+// logical lane segments (green, blue and red).
+//
+message Route
+{
+
+    // Route segments that form the planned route of an agent.
+    //
+    // Consecutive segments should be connected without gaps, meaning that the
+    // two of them should form a continuous area.
+    //
+    repeated RouteSegment planned_route = 1;
```

Comment:
WG Other Models: route_segment
Plural is added automated by generator.

Reply:
I adjusted the message as you suggested.
Regarding the feedback from the WG:

- I'm not sure if the route message has to match OpenScenario. At least for the use case where it originated from, this was not required.
- I added some additional information on the purpose and benefits of the route message to the documentation.
- No